theano/gpuarray/opt.py

```diff
@@ -55,14 +59,34 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
 gpu_cut_copies = EquilibriumDB()
 
+old_not_transferred = []
+new_not_transferred = []
```

Comment:
It is useful information for debugging, then, but not in the final version of the PR.